### DISCUSSION TITLE
MacBook 键盘效率工具推荐 | SF-Zhou's Blog

### DISCUSSION THREAD

**Discussion by SF-Zhou** [SF-Zhou/sf-zhou.github.io#154] [Announcements]
https://sf-zhou.github.io/productivity/macbook_keyboard_tools.html

**Comment by limj0825** [comment]
vscode有没有什么好用的插件推荐？

**Reply by aircloud** [reply]
借SF大佬的楼，推荐几个

* Dictionary Completion: 字典补全
* Project Manager 非常方便地切换多个项目。
* Blockman 用浅色背景高亮一个代码块，比如一个 if 块
* Bookmarks 针对某行加书签
* todo tree 建议和 todo highlighter 一起使用，可以侧边栏管理你的 TODO
* Bracket Pair Colorization 可以自动给匹配的括号加上不同的颜色，让我们更方便地找到括号对应关系。

**Comment by Flyingrhino007** [comment]
Mac上Clipy挺好用的，推荐

**Comment by godhangyu** [comment]
窗口工具推荐rectangle 不妨试试看 感觉更傻瓜

**Reply by SF-Zhou** [reply]
看起来挺好用，不过我自己习惯了就不换了 :smile:

**Reply by SF-Zhou** [reply]
链接：https://github.com/rxhanson/Rectangle
